src/components/dynamic/admin/Table.jsx

```diff
@@ -94,7 +94,7 @@ const Table = ({
                   object.selected ? "bg-green-100" : "bg-white"
                 }`}
               >
-                <div className="flex justify-between items-center">
+                <div className="flex justify-start items-center">
```

Comment:
this change should not be necessary

Reply:
But without it, it looks like this:
<img width="568" alt="image" src="https://github.com/acm-ucr/hackathon-website/assets/64669569/bca89a41-6264-47cc-a837-ba05ba7a4c34">

I initially thought so too and was confused, but Menthy suggested I add justify-start to accordion and then that worked.